test_go_pricing_page_by_click_pricing_btn (__main__.Test_Chatbot.test_go_pricing_page_by_click_pricing_btn) ... ok
test_go_pricing_page_through_chatbot_keyboard (__main__.Test_Chatbot.test_go_pricing_page_through_chatbot_keyboard) ... ok
test_invalid_input (__main__.Test_Chatbot.test_invalid_input) ... FAIL
test_multi_language (__main__.Test_Chatbot.test_multi_language) ... FAIL
test_similar_word (__main__.Test_Chatbot.test_similar_word) ... FAIL

======================================================================
FAIL: test_invalid_input (__main__.Test_Chatbot.test_invalid_input)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "c:\Users\kit\Documents\GitHub\Onramp-Lab-QA-Jack\QA_Trial_Test\Test_Chatbot.py", line 123, in test_invalid_input
    self.assertEqual(chatbot_response, "üòµ‚Äçüí´ Oops! Sorry, I didn't understand your question. Please rephrase it or click the button below to go back to the menu.")
AssertionError: 'What brought you here today?' != "üòµ\u200düí´ Oops! Sorry, I didn't understan[81 chars]enu."
- What brought you here today?
+ üòµ‚Äçüí´ Oops! Sorry, I didn't understand your question. Please rephrase it or click the button below to go back to the menu.


======================================================================
FAIL: test_multi_language (__main__.Test_Chatbot.test_multi_language)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "c:\Users\kit\Documents\GitHub\Onramp-Lab-QA-Jack\QA_Trial_Test\Test_Chatbot.py", line 134, in test_multi_language
    pricing_btn = self.driver.find_element(By.CSS_SELECTOR, ".quick-replies-buttons .single-button:nth-child(2)")
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\kit\AppData\Local\Programs\Python\Python311\Lib\site-packages\selenium\webdriver\remote\webdriver.py", line 740, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\kit\AppData\Local\Programs\Python\Python311\Lib\site-packages\selenium\webdriver\remote\webdriver.py", line 346, in execute
    self.error_handler.check_response(response)
  File "C:\Users\kit\AppData\Local\Programs\Python\Python311\Lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 245, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"css selector","selector":".quick-replies-buttons .single-button:nth-child(2)"}
  (Session info: chrome=115.0.5790.110); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
Backtrace:
	GetHandleVerifier [0x00741C53+49427]
	(No symbol) [0x006D5F41]
	(No symbol) [0x005DC64D]
	(No symbol) [0x00609818]
	(No symbol) [0x006098DB]
	(No symbol) [0x006387D2]
	(No symbol) [0x00624A64]
	(No symbol) [0x00636F2A]
	(No symbol) [0x00624816]
	(No symbol) [0x00601127]
	(No symbol) [0x006022AD]
	GetHandleVerifier [0x00997149+2496009]
	GetHandleVerifier [0x009DD572+2783794]
	GetHandleVerifier [0x009D7491+2758993]
	GetHandleVerifier [0x007C11D0+571024]
	(No symbol) [0x006DF96A]
	(No symbol) [0x006DBD88]
	(No symbol) [0x006DBE6B]
	(No symbol) [0x006CEA97]
	BaseThreadInitThunk [0x76EA00C9+25]
	RtlGetAppContainerNamedObjectPath [0x77077B1E+286]
	RtlGetAppContainerNamedObjectPath [0x77077AEE+238]


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\Users\kit\Documents\GitHub\Onramp-Lab-QA-Jack\QA_Trial_Test\Test_Chatbot.py", line 156, in test_multi_language
    raise AssertionError("An error occurred during the test: %s" % str(e))
AssertionError: An error occurred during the test: Message: no such element: Unable to locate element: {"method":"css selector","selector":".quick-replies-buttons .single-button:nth-child(2)"}
  (Session info: chrome=115.0.5790.110); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
Backtrace:
	GetHandleVerifier [0x00741C53+49427]
	(No symbol) [0x006D5F41]
	(No symbol) [0x005DC64D]
	(No symbol) [0x00609818]
	(No symbol) [0x006098DB]
	(No symbol) [0x006387D2]
	(No symbol) [0x00624A64]
	(No symbol) [0x00636F2A]
	(No symbol) [0x00624816]
	(No symbol) [0x00601127]
	(No symbol) [0x006022AD]
	GetHandleVerifier [0x00997149+2496009]
	GetHandleVerifier [0x009DD572+2783794]
	GetHandleVerifier [0x009D7491+2758993]
	GetHandleVerifier [0x007C11D0+571024]
	(No symbol) [0x006DF96A]
	(No symbol) [0x006DBD88]
	(No symbol) [0x006DBE6B]
	(No symbol) [0x006CEA97]
	BaseThreadInitThunk [0x76EA00C9+25]
	RtlGetAppContainerNamedObjectPath [0x77077B1E+286]
	RtlGetAppContainerNamedObjectPath [0x77077AEE+238]


======================================================================
FAIL: test_similar_word (__main__.Test_Chatbot.test_similar_word)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "c:\Users\kit\Documents\GitHub\Onramp-Lab-QA-Jack\QA_Trial_Test\Test_Chatbot.py", line 182, in test_similar_word
    self.assertTrue(self.all_words_passed, "One of the word failed")
AssertionError: False is not true : One of the word failed

----------------------------------------------------------------------
Ran 5 tests in 386.830s

FAILED (failures=3)
